app/src/main/res/layout/layout_listitem.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/colorAccent1"
+    android:padding="15dp"
+    android:id="@+id/parent_layout">
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        app:cardBackgroundColor="#cdcdcd"
+
+        android:layout_height="match_parent"
+        app:cardCornerRadius="16dp">
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        app:cardBackgroundColor="#f2f2f2"
+        app:cardUseCompatPadding="true"
+        android:layout_height="match_parent"
+        app:cardElevation="6dp"
+        android:layout_margin="5dp"
+        app:cardCornerRadius="16dp">
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        app:cardElevation="6dp"
+        app:cardBackgroundColor="#ffffff"
+        app:cardCornerRadius="16dp"
+        android:layout_margin="5dp"
+        app:cardUseCompatPadding="true">
+
+
+
+
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Canada"
+            android:textStyle="bold"
+            android:padding="15dp"
+            android:id="@+id/image_name"
+            android:layout_marginTop="15dp"
+            android:textColor="#ffffff"
+            android:background="@drawable/rounded_corner"
+            android:layout_gravity="center_horizontal"
+            android:textSize="35sp"
+            />
+
+
+    <com.makeramen.roundedimageview.RoundedImageView
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/image"
+        android:src="@mipmap/ic_launcher"
+        android:layout_height="300dp"
+        android:layout_width="300dp"
+        android:layout_gravity="center_horizontal"
+        android:scaleType="fitCenter"
+        app:riv_corner_radius="30dip"
+
+
+        />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="               If you're working with an existing view it can be a lot of work creating a new set of LayoutParams from scratch. Instead - you can grab the view's existing LayoutParams, edit these, and then apply them to the view to update its LayoutParams using setLayoutParams()"
```

Comment:
Try to add the Text in  String.xml File and refer to those strings .

Reply:
Its just a sample string.Does nothing.So, now I just changed it to one word "description"